src/Pact/Analyze/Parse/Prop.hs

```diff
@@ -145,8 +145,6 @@ expToPreProp = \case
     pretty SPropRead <> " must specify a time ('before or 'after). example: " <>
     "(= result (read accounts user 'before))"
 
-  exp@(ParenList [EAtom' SObjectProjection, _, _]) -> throwErrorIn exp
-    "Property object access must use a static string or symbol"
```

Comment:
Everything in here looks good, except this one. ~one of these things is not like the others~


Could you expand as to why this needs to be removed?

Reply:
This pattern match is redundant. The case is covered a few lines above by:

```Haskell
-- Note: this handles both object and list projection:
ParenList [EAtom' SObjectProjection, ix, container]
  -> PreAt <$> expToPreProp ix <*> expToPreProp container
```